src/DistributedWriteAheadLog/KafkaWALSettings.h

```diff
@@ -58,7 +58,6 @@ struct KafkaWALSettings
     int32_t session_timeout_ms = 10000;
     int32_t max_poll_interval_ms = 30000;
 
-    bool enable_auto_commit = true;
```

Comment:
We can delete this setting, but we can't delete line 97.

Reply:
fixed